app/src/main/java/org/keepgoeat/presentation/onboarding/OnBoardingActivity.kt

```diff
@@ -0,0 +1,62 @@
+package org.keepgoeat.presentation.onboarding
+
+import android.os.Bundle
+import androidx.activity.viewModels
+import androidx.viewpager2.widget.ViewPager2
+import com.google.android.material.tabs.TabLayoutMediator
+import dagger.hilt.android.AndroidEntryPoint
+import org.keepgoeat.R
+import org.keepgoeat.databinding.ActivityOnboardingBinding
+import org.keepgoeat.util.binding.BindingActivity
+
+@AndroidEntryPoint
+class OnBoardingActivity : BindingActivity<ActivityOnboardingBinding>(R.layout.activity_onboarding) {
+    private val viewModel: OnBoardingViewModel by viewModels()
+    private val mockOnboardingList = listOf(
+        OnBoardingItem(
+            title = R.string.onboarding1_title,
+            des = R.string.onboarding1_des,
+            image = R.drawable.img_onboarding_1
+        ),
+        OnBoardingItem(
+            title = R.string.onboarding2_title,
+            des = R.string.onboarding2_des,
+            image = R.drawable.img_onboarding_2
+        ),
+        OnBoardingItem(
+            R.string.onboarding3_title,
+            des = R.string.onboarding3_des,
+            image = R.drawable.img_onboarding_3
+        )
+    )
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding.viewModel = viewModel
+        binding.lifecycleOwner = this
+        initLayout()
+        addListeners()
+    }
+
+    private fun initLayout() {
+        val adapter = OnBoardingAdapter(this)
+        with(binding) {
+            vpViewPager.isUserInputEnabled = false
+            vpViewPager.adapter = adapter
+            adapter.setOnBoardingList(mockOnboardingList)
+            vpViewPager.registerOnPageChangeCallback(object : ViewPager2.OnPageChangeCallback() {
+                override fun onPageSelected(position: Int) {
+                    super.onPageSelected(position)
+                    viewModel?.setPosition(position)
+                }
+            })
+            TabLayoutMediator(binding.indicator, vpViewPager) { tab, position -> }.attach()
```

Comment:
사용되지 않는 변수네용! 변수명을 _로 표시해봅시다~

```suggestion
            TabLayoutMediator(binding.indicator, vpViewPager) { _, _ -> }.attach()
```